workload/bicep/deploy-baseline.bicep

```diff
@@ -171,8 +171,8 @@ param msixFileShareQuotaSize int = 1
 @sys.description('Deploy new session hosts. (Default: true)')
 param avdDeploySessionHosts bool = true
 
-@sys.description('Deploy VM GPU extension policies. (Default: true)')
-param deployGpuPolicies bool = true
+@sys.description('Deploy VM GPU extension policies. (Default: false)')
+param deployGpuPolicies bool = false
```

Comment:
Didn't see this change in your PR body, just wanted to make sure it was intentional and not a temporary/local change you did for testing